### ISSUE TITLE
unschedule(nil) - unschedules the first scheduled job

### ISSUE THREAD

**Issue by david-a** [utgarda/sidekiq-status#17]
Sidekiq::Status.unschedule(nil) unschedules the first scheduled job (no matter what it is). Unless I'm missing something here, I think the wanted behaviour is to just return nil without acting (or even returning an error).

**Comment by jmnsf** [comment]
Lost while hunting this bug, +1 on a fix.

**Comment by andyryan** [comment]
Literally came here to see if anyone had already submitted this bug. Nice one

**Comment by pramodshinde** [comment]
@utgarda - Fixed this issue in PR #69.
Problem was with following [line](https://github.com/utgarda/sidekiq-status/blob/master/lib/sidekiq-status/storage.rb#L94) 
```ruby
scheduled_jobs.each { |job_listing| (return job_listing) if job_listing.include?("\"jid\":\"#{job_id}") }
```
Above line was checking patterns inclusion only, so following was getting true if `unschedule(nil)`
Ex: job_listing
```ruby
"{\"retry\":false,\"queue\":\"default\",\"backtrace\":true,\"expires_in\":86400,\"class\":\"MyWorker\",\"args\":[10],\"jid\":\"a28ba4cc9b0396da3fb2c410\",\"enqueued_at\":1449306022.4224896}" 
```
```ruby
job_listing.include?("\"jid\":\"#{nil}")
=> \"jid\":\" 
# resulting in true as \"jid\":\"a28ba4cc9b0396da3fb2c410\" has inclusion of \"jid\":\"
```
Replaced inclusion check with [this](https://github.com/joshsoftware/sidekiq-status/blob/b52e2e2e3cff070aa8333470c869da0506108d98/lib/sidekiq-status/storage.rb#L94), Additional I have tested this change and it will handle all [cases](https://github.com/joshsoftware/sidekiq-status/commit/b52e2e2e3cff070aa8333470c869da0506108d98#diff-04c6e90faac2675aa89e2176d2eec7d8R124) like `unschedule(nil)`, `unschedule('')` and `unschedule(other_jid)`.
